build.gradle.kts

```diff
@@ -232,6 +232,30 @@ subprojects {
     }
 
     apply(plugin = "gradlebuild.test-files-cleanup")
+
+    plugins.withType<JavaPlugin> {
```

Comment:
Maybe add a little comment there that we introduced this new configuration so that the Tooling API sources jar file can be assembled correctly (i.e. the jar contains sources from all transitive dependencies).